roles/check_local_time/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
+
+  block:
+    - name: Get Time Zone
+      scale_computing.hypercore.time_zone_info:
+      register: time_zone
+
+    - name: Check if local time is in time interval (run local_time.py)
+      ansible.builtin.script:
+        executable: python
```

Comment:
Potential problem, on Ubuntu we might have python3 , but no python.
Test
```
docker run -it -w /code -v $PWD:/code ubuntu:latest
apt-get update
apt-get install -y python3
  # we have python3 now

apt-get install -y python
  # no such thing, but in mentions you might want to install python-is-python3
```

Alternatives:
- use python3 - but you must test it will be accessible on major distributions
- refactor code to be invoked as a module (a module we want to have for internal use only, hm)
- you can check which python3 or python is on path (`which python3` or `which python`). I sort of prefer that one.